### ISSUE TITLE
Inquiry: Angular feature parity roadmap

### ISSUE THREAD

**Issue by shlomiassaf** [NativeScript/nativescript-angular#391]
Hi,

First, Thank you for the great/awesome/superb/your-pick work!

I've been using Angular 2 for a long time, since the early alphas, updating my apps as change comes.

With the latest release of RC5 its seems that breaking API changes (not to say concepts...) will not happen, at least not major ones.

Currently (using `@next` packages) angular and nativescript are at RC4 and Router alpha 2.
This is quite outdated, especially at the router part and `@NgModule` in RC5.

Looking at the milestones I didn't see any reference or a roadmap referring sync with Angular 2.

I understand that keeping up with the changes in Angular is a big deal, this is why a roadmap is super important so developers can plan ahead and decide accordingly.

Where can one view the angular version parity roadmap?

Thanks, Shlomi

**Comment by valentinstoychev** [comment]
hey Shlomi, what do you mean as a roadmap? We basically cover the entire stack of functionality of the Angular 2 APIs. What do you think it is missing in NativeScript that we can add?

As for the RC5 compatibility - we need some time to update our SDK to this since there are tons of changes that affect us badly, but we are actively working on it with top priority for us. RC5 was just out yesterday and usually it takes us 1-2 weeks to catch up on the Angular releases since many of the things may seems like small things from the outside, but can affect us tremendously since we are working with many of the internals of the Angular SDKs ...

Please let us know what are you missing and we will get back with a more detailed answer. 

Thank for looking into NativeScript!

**Comment by shlomiassaf** [comment]
Hi @valentinstoychev. 
Thank you for the detailed response.

Basically, you answered what I wanted to know,

When I looked at the [milestones](https://github.com/NativeScript/nativescript-angular/milestone/8) for 2.3 I did not see any reference for feature sync with RC4 nor RC5.

Given then RC4 is out for a long time I assumed you wouldn't work on it since RC5 has some major concept changes. But since I didn't see any reference I thought it's out of scope for the upcoming months...

I understand that it takes some weeks to fill the gap, especially with conceptual changes the angular team tends to do rapidly (RC? alpha maybe....)

Anyway, It would be great having a milestone for angular RC5 sync and router sync.

Thanks again, for the answer and for the great work!
